### PR TITLE
Add antimeridian check for INSAR_ISCE_BURST V2

### PR DESCRIPTION
**Pull request by forrestfwilliams**

### CODE REVIEW

File: apps/api/src/hyp3_api/validation.py [hunk 1]
```diff
@@ -93,6 +93,13 @@ def check_valid_polarizations(granule_metadata):
         raise GranuleValidationError(f'Only VV and HH polarizations are currently supported, got: {ref_polarization}')
 
 
+def check_not_antimeridian(granule_metadata):
+    for granule in granule_metadata:
+        bbox = granule['polygon'].bounds
+        if abs(bbox[0] - bbox[2]) > 180.0 and bbox[0] * bbox[2] < 0.0:
+            raise GranuleValidationError(f'Granule {granule["name"]} crosses the antimeridian')
```
Comment (asjohnston-asf) [inline apps/api/src/hyp3_api/validation.py:100]: A slightly more explicit error message would be worthwhile, e.g. "Granule {} crossing the antimeridian is not currently supported"

File: tests/test_api/test_validation.py [hunk 2]
```diff
@@ -10,6 +10,21 @@ def rectangle(north, south, east, west):
     return Polygon([[west, north], [east, north], [east, south], [west, south]])
 
 
+def test_not_antimeridian():
+    # Wyoming
+    good_rect = rectangle(45, 41, -104, -111)
+
+    # completely covered Aleutian Islands over antimeridian; should fail
```
Comment (asjohnston-asf) [inline tests/test_api/test_validation.py:17]:
```suggestion
    # Aleutian Islands over antimeridian; should fail
```

**Comment by forrestfwilliams** [comment]
Ready for re-review!

File: apps/api/src/hyp3_api/validation.py (hunk 1 above)
Comment (forrestfwilliams) [reply apps/api/src/hyp3_api/validation.py]: Hopefully the new version works for you.

**Review by jtherrmann — APPROVED** [review]
LGTM!

**Pull request merged**